__tests__/components/editor/ResourceTemplateForm.test.js

```diff
@@ -15,7 +15,10 @@ const rtProps = {
     },
     {
       "propertyLabel": "Look up, look down",
-      "type": "lookup"
+      "type": "lookup",
+      "editable": "do not override me!",
+      "repeatable": "do not override me!",
+      "mandatory": "do not override me!"
```

Comment:
FYI, Christina and I would like to change these values over to boolean (instead of strings) in our JSON schema ... but that would also imply a change to profile editor code ...